src/Toolbox.php

```diff
@@ -3539,7 +3539,7 @@ public static function isValidWebUrl($url): bool
        // It is still possible to have false positives, but a fireproof check would be too complex
        // (or would require usage of a dedicated lib).
         return (preg_match(
-            "/^(?:http[s]?:\/\/(?:[^\s`!(){};'\",<>«»“”‘’+]+|[^\s`!()\[\]{};:'\".,<>?«»“”‘’+]))$/iu",
```

Comment:
regexps are always quite hard to read, but would'nt this change allows too much (including characters that should not be present in an URL)?